ci/actions-templates/linux-builds-template.yaml

```diff
@@ -169,6 +169,18 @@ jobs: # skip-master skip-pr skip-stable
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           AWS_DEFAULT_REGION: us-west-1
+      - name: Configure AWS credentials
+        if: github.event_name == 'push' && github.ref == 'refs/heads/master' && matrix.mode == 'release'
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: arn:aws:iam::890664054962:role/ci--rust-lang--rustup
+          aws-region: us-east-1
+      - name: Deploy build to rustup-builds bucket for release team
+        if: github.event_name == 'push' && github.ref == 'refs/heads/master' && matrix.mode == 'release'
+        run: |
+          aws --debug s3 cp --recursive deploy/ s3://rustup-builds/${{ github.sha }}
```

Comment:
There has been a similar `aws s3 cp --recursive deploy/ s3://dev-static-rust-lang-org/rustup/` command right above. 
I guess this is intentional, in that you want to make additive changes for now and remove the old one only when you're sure the new approach works?

Reply:
Oh yeah, sorry! I forgot to mention this above. The changes right now are all additive as to not break the currently release process. Once the new one has been implemented and tested, I'll clean up the old steps.